sparse/core.py

```diff
@@ -912,3 +1284,48 @@ def stack(arrays, axis=0):
 
     return COO(coords, data, shape=shape, has_duplicates=has_duplicates,
                sorted=(axis == 0) and all(a.sorted for a in arrays))
+
+
+# (c) Paul Panzer
+# Taken from https://stackoverflow.com/a/47833496/774273
+# License: https://creativecommons.org/licenses/by-sa/3.0/
```

Comment:
There was a nice bit of conversation on this question.  Looked like a very productive use of Stack Overflow :)